.travis.yml

```diff
@@ -30,6 +30,14 @@ script:
       bzr branch lp:govcstestbzrrepo;
       hg clone http://www.selenic.com/repo/hello;
     "
+  # Verify MarkdownBear works outside of coala-bears directory.
+  # See https://github.com/coala/coala-bears/issues/1235
+  - >
+    docker run -t -i coala-docker /bin/sh -c "
+      cd /tmp;
+      echo foo > foo.md;
+      coala-ci -I --bears MarkdownBear --files foo.md;
```

Comment:
Isn't `coala-ci` deprecated? I think it should be `coala --non-interactive` instead.